man/man8/zpool.8

```diff
@@ -1697,6 +1709,28 @@ Display vdev GUIDs instead of the normal device names. These GUIDs can be used i
 .ne 2
 .mk
 .na
+\fB\fB-h\fR\fR
+.ad
+.RS 12n
+.rt
+Display help (usage)
+.RE
```

Comment:
I'd consider dropping this from the man page to be consistent with all the other sub-commands.  It's OK that it work but I think it's a little distracting to add here with all the other iostat specific commands.